src/App.css

```diff
@@ -26,3 +22,20 @@
   from { transform: rotate(0deg); }
   to { transform: rotate(360deg); }
 }
+
+
+/* Styles */
+.container {
```

Comment:
you shouldn't use css, everything should be migrated to styled components

Reply:
Yep, I know Tim. I'm working right now to move the styles.

Thank you!!